tutorons/css/explain.py

```diff
@@ -24,15 +29,43 @@
 def explain(selector):
     return explainer.explain(selector)
 
```

Comment:
Spacing is inconsistent with the rest of the file.
Check to make sure that formatting conforms with PEP8 standards:

    pip install flake8
    flake8 --max-line-length=100 tutorons/css/

Then make sure to solve any problems introduced by your changes.